dbaas/backup/tasks.py

```diff
@@ -130,7 +136,23 @@ def make_instance_snapshot_backup(instance, error, group,
                     "Backup with WARNING already created today."
                     )
         else:
-            response = provider.take_snapshot()
+            for _ in range(backup_retry_attempts):
```

Comment:
I think is better replace this for with a while statement, as we does not use the "_" variable, eg:

```python
counter = 0
while counter < range(backup_retry_attempts):
    counter += 1[...]```